src/test/javascript/spec/component/account/vcs-access-token-settings.component.spec.ts

```diff
@@ -0,0 +1,143 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { AccountService } from 'app/core/auth/account.service';
+import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
+import { of } from 'rxjs';
+import { By } from '@angular/platform-browser';
+import { MockComponent, MockPipe } from 'ng-mocks';
+import { ButtonComponent } from 'app/shared/components/button.component';
+import { FormDateTimePickerComponent } from 'app/shared/date-time-picker/date-time-picker.component';
+import { User } from 'app/core/user/user.model';
+import { ArtemisTestModule } from '../../test.module';
+import { MockNgbModalService } from '../../helpers/mocks/service/mock-ngb-modal.service';
+import { MockTranslateService, TranslatePipeMock } from '../../helpers/mocks/service/mock-translate.service';
+import { TranslateService } from '@ngx-translate/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { PROFILE_LOCALVC } from 'app/app.constants';
+import { VcsAccessTokensSettingsComponent } from 'app/shared/user-settings/vcs-access-tokens-settings/vcs-access-tokens-settings.component';
+import dayjs from 'dayjs';
```

Comment:
This should fix the failing client test build:

```suggestion
import dayjs from 'dayjs/esm';
```